package/kernel/linux/modules/crypto.mk

```diff
@@ -655,7 +748,6 @@ endef
 
 $(eval $(call KernelPackage,crypto-test))
 
-
```

Comment:
Please avoid unnecessary whitespace changes.

Reply:
These were accidental through Github's web interface, sorry. I know there is a way to squash the commits into the first one.